### DISCUSSION TITLE
Treeshaking Next.js

### DISCUSSION THREAD

**Discussion by The-Sirius-Black** [lovrozagar/renderui-core#3] [Q&A] [answered]
Hey thanks for fast reply i saw your answer on issue i opened.
I still want to know specifically for Next.js if i need to use the optimizePackageImports in the next config to enable tree shaking?

**Comment by lovrozagar** [comment, marked as answer]
Hi @The-Sirius-Black,

The library takes care of tree shaking out of the box by splitting the .js modules in build time so this setting should not be required. 
However you can enable it if you wish, it should not have negative side effects, as far as the library is concerned.

I hope this helps!
